src/components/Tabs/components/Tab/Tab.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import Element from '../../../Element';
+
+import { textMega } from '../../../../styles/style-helpers';
+
+const defaultTabStyles = ({ theme }) => css`
+  label: tab;
+  ${textMega({ theme })};
+  padding: ${theme.spacings.kilo} ${theme.spacings.tera};
+  color: ${theme.colors.n500};
+  text-decoration: none;
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+
+  white-space: nowrap;
+  height: 100%;
+  align-items: center;
+  float: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  outline: none;
+`;
+
+const selectedTabStyles = ({ theme, selected }) =>
+  selected &&
+  css`
+    label: tab--selected;
+    color: initial !important;
+    position: relative;
+
+    ::after {
+      content: ' ';
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      width: 100%;
+      height: ${theme.spacings.bit};
+      background: ${theme.colors.p500};
+    }
+  `;
+
+const tabIndex = selected => (selected ? undefined : '-1');
```

Comment:
Why are you setting it to `-1`? This removes the tab from the focusable elements on the page 🧐

Reply:
I saw it being used in other implementations. The end result is that when you click a tab, the next tab element is inside the panel.

Reply:
Did you mean to write...
```suggestion
const tabIndex = selected => (selected ? '-1' : undefined);
```

Then it would be focusable by default, and likely pass focus to the next focusable element when selected (not sure if that actually works, I'd be curious to test that with a screen reader).

Reply:
I spent a bit more time reading about the topic because I want to understand this accessibility pattern better and because I don't want to give bad advice based on my incomplete assumptions. 

Once again, I found a lot of helpful information in one of [Heydon Pickering's articles](https://inclusive-components.design/tabbed-interfaces/). He confirms that what you are doing here is the best thing for screen readers.

> It's equally important that pressing `Shift + Tab` returns the user to the selected tab. This is all possible by giving each tab but the selected tab `tabindex="-1"`, which removes the inactive tabs from focus order but allows focus via a script. In the following example, the second tab is the selected tab, as denoted by the `aria-selected` state being set to `true`.
> 
> [...]
> 
> With `tabindex="-1`" in place for the remaining tabs, I can capture the `keydown` event for the left or right arrow keys to make the desired inactive tab active.

Based on this description, it seems like you're missing a bit of manual focus management to make the tabs fully screen reader accessible. I strongly recommend reading [this relevant section](https://inclusive-components.design/tabbed-interfaces/#aproblemreadingpanels) in Heydon's article.